src/main/java/frc/robot/Constants.java

```diff
@@ -56,22 +62,17 @@ public static final class Swerve {
 
         public static final PIDConfig autoRotate = PIDConfig.getPid(0.1);
 
-         public static final boolean SWERVE_TUNNING_MODE = true;
+        public static final boolean SWERVE_TUNNING_MODE = true;
+        public static final SDSModules MODULE_TYPE = SDSModules.MK4i;
```

Comment:
nit: I don't think you needed to break this out into a constant unless you are referencing it somewhere else, in which case ignore this